server/routerlicious/packages/services-ordering-rdkafka/src/rdkafkaConsumer.ts

```diff
@@ -263,6 +267,10 @@ export class RdkafkaConsumer extends RdkafkaBase implements IConsumer {
 			this.emit("throttled", event);
 		});
 
+		consumer.on("event.log", (event) => {
+			this.emit("log", event);
```

Comment:
How did you consume these `log` events? Did you add custom listeners on your RdKafkaConsumer and RdKafkaProducer objects?

Reply:
Yea. These log events are emitted when rdkafka debug flags are set